downstream/modules/platform/ref-operator-ocp-version.adoc

```diff
@@ -4,8 +4,6 @@
 
 [role="_abstract"]
 
-The {OperatorPlatform} to install {PlatformNameShort} {PlatformVers} is available on {OCPShort} 4.9 and later versions.
+The {OperatorPlatform} to install {PlatformNameShort} {PlatformVers} is available on {OCPShort} 4.10 to 4.13.
 
 [role="_additional-resources"]
-.Additional resources
```

Comment:
If you're removing the additional resources section, be sure to remove [role="_additional-resources"] otherwise the formatting will get weird.

Reply:
did this! thanks